wtransport/src/driver/utils.rs

```diff
@@ -25,9 +25,9 @@ pub fn varint_w2q(varint: VarInt) -> quinn::VarInt {
 
 #[inline(always)]
 pub fn streamid_q2w(stream_id: quinn::StreamId) -> StreamId {
-    let varint = unsafe {
-        debug_assert!(u64::from(stream_id) <= VarInt::MAX.into_inner());
-        VarInt::from_u64_unchecked(u64::from(stream_id))
+    let varint: wtransport_proto::varint::VarInt = unsafe {
+        debug_assert!(quinn::VarInt::from(stream_id) <= quinn::VarInt::MAX);
```

Comment:
Why changing the meaning of the assert... we are not validating `quinn` code, but our unsafe.

```
debug_assert!(quinn::VarInt::from(stream_id).into_inner() <= VarInt::MAX.into_inner())
```